DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
Yes.

Currently the eras lines do nothing. They are a work in progress.

The way this is currently done in production workflows is via

https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py#L134